src/Account/components/AccountTransactions.tsx

```diff
@@ -3,17 +3,21 @@ import { useTranslation } from "react-i18next"
 import DoneAllIcon from "@material-ui/icons/DoneAll"
 import CreditCardIcon from "@material-ui/icons/CreditCard"
 import UpdateIcon from "@material-ui/icons/Update"
-import { Account } from "../../context/accounts"
-import { SettingsContext } from "../../context/settings"
-import { SignatureDelegationContext } from "../../context/signatureDelegation"
-import { hasSigned } from "../../lib/transaction"
-import { useHorizonURL } from "../../hooks/stellar"
-import { useLiveRecentTransactions, useLiveAccountData, useOlderTransactions } from "../../hooks/stellar-subscriptions"
-import { useIsMobile, useRouter } from "../../hooks/userinterface"
-import { useLoadingState } from "../../hooks/util"
+import { Account } from "~App/contexts/accounts"
+import { SettingsContext } from "~App/contexts/settings"
+import { SignatureDelegationContext } from "~App/contexts/signatureDelegation"
+import { hasSigned } from "~Generic/lib/transaction"
+import { useHorizonURL } from "~Generic/hooks/stellar"
+import {
+  useLiveRecentTransactions,
+  useLiveAccountData,
+  useOlderTransactions
+} from "~Generic/hooks/stellar-subscriptions"
+import { useIsMobile, useRouter } from "~Generic/hooks/userinterface"
+import { useLoadingState } from "~Generic/hooks/util"
 import * as routes from "../../routes"
```

Comment:
Is there a reason why we don't do this for routes? We could just use `import * as routes from "~/routes"` everywhere right?

Reply:
Yep, should work. The only reason is really that I used some regex search & replace and it wasn't matched by the expression 😉